ibm/resource_ibm_atracker_route.go

```diff
@@ -0,0 +1,246 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/platform-services-go-sdk/atrackerv1"
+)
+
+func resourceIBMAtrackerRoute() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMAtrackerRouteCreate,
+		ReadContext:   resourceIBMAtrackerRouteRead,
+		UpdateContext: resourceIBMAtrackerRouteUpdate,
+		DeleteContext: resourceIBMAtrackerRouteDelete,
+		Importer:      &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
We don't have any wait logic in the code so we may not need timeouts...
or else do we have API to see the status of route something like (creating, attaching,..available...so on) ...and needs any wait logic until the desired state?

Reply:
Removed timeouts in ATracker.yaml and regenerated code.